gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-info.spec.ts

```diff
@@ -18,7 +18,7 @@ import { ApisFaker } from '@gravitee/fixtures/management/ApisFaker';
 import { Visibility } from '@gravitee/management-webclient-sdk/src/lib/models';
 import faker from '@faker-js/faker';
 import ApiDetails from 'ui-test/support/PageObjects/Apis/ApiDetails';
-import { ApiImport } from '@model/api-imports';
+import { Api } from '@model/apis';
```

Comment:
I would suggest keeping the models separate because there is an added value in re-analyzing the objects on the e2e side. Additionally, if we reference the models in `gravitee-apim-console-webui`, we increase the dependency between e2e and the code base. I would recommend keeping the modules as separate as possible in order to maximize the value of the test environment.

Reply:
I see in the git blame that @ytvnr has worked on the Api model in e2e tests. Maybe you have something to add to this topic?

Reply:
sorry but both links provided by @ctschacher lead to a 404
from what I see the work on Api model in e2e is pretty old, so I think it was just the first step (before having the clients and models generated from swagger)

Imho, as we are generating model from swagger, the easiest and safest would be to use these types (as done for api e2e test in jest, in `lib/management-webclient-sdk`) instead of manually created interface. I strongly agree with @jourdiw about the fact we should avoid a direct dependency to the console project.